### PR TITLE
Full Website

### PR DESCRIPTION
**Pull request by 41hulk**
### What does this PR do?
- [ ] Create a website consisting of 4 pages: Home, Calculator, and Quote & Not Match.
- [ ] Styles all the pages using React basic styling.
- [ ] Create a route for every page.
- [ ] Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/pages/error.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NotMatch = () => (
+  <div className="no-match">
+    <h3>Whoops! Page not found!</h3>
+  </div>
+);
+
+export default NotMatch;
```
Comment (Gambit142) [inline src/pages/error.js:9]: - [ ] Nice job rendering a page when the wrong `URL` endpoint is inputted. Kudos 👏

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @41hulk 👋, 

Good job so far!

### Highlights

- The project works as it should ✔️ 
- Proper use of `react-router-dom` ✔️ 
- All linters checks passed ✔️ 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Kindly update your `README.md` file to a more professional one by customizing the Headers.  You can use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a guide. Your current `README.md` file is that which comes with the `CRA` command. You can also decide to use some of the commands from the `CRA` README.md files like how to install, run build, run test, start the server.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Gambit142 — APPROVED** [review]
Hi @41hulk 👋, 

Great work implementing the change. 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 41hulk** [comment]
Thanks @Gambit142

**Pull request merged**